lib/model/util.go

```diff
@@ -59,3 +68,95 @@ func (d *deadlockDetector) Watch(name string, mut sync.Locker) {
 		}
 	}()
 }
+
+// moveforconflict renames a file to deal with sync conflicts
+func moveForConflict(name string, maxConflicts int) error {
```

Comment:
This function is used in rwfolder also, that's why I moved it to util.go. Are you suggesting to move it to folder.go instead?